services/src/main/scala/cromwell/services/metadata/impl/ReadDatabaseMetadataWorkerActor.scala

```diff
@@ -66,6 +68,15 @@ class ReadDatabaseMetadataWorkerActor(metadataReadTimeout: Duration, metadataRea
     }
   }
 
+  private def getMetadataStream(query: MetadataQuery, fetchSize: Int): Try[MetadataServiceResponse] = {
+    metadataEventsStream(query, fetchSize, metadataReadTimeout) map {
+      s => MetadataLookupStreamResponse(query, s)
+    } recover {
+      case _: SQLTimeoutException => MetadataLookupFailedTimeoutResponse(query)
```

Comment:
Probably just overly-paranoid copy paste. I assume this goes away when I remove the unused `timeout` parameter.

As far as I can tell, getting a stream is not something that times out (presumably because it doesn't send the first SQL command until we first try reading from it)